README.md

```diff
@@ -72,6 +72,16 @@ You can change the path of the VCL configuration file:
 $ docker run --name my-running-varnish -e "VCL_CONFIG=/root/custom.vcl" -v /path/to/custom.vcl:/root/custom.vcl:ro -d my-varnish
 ```
 
+You can also change the ports used in the Dockerfile.
+
+```
+FROM tripviss/varnish
+
+ENV VARNISH_PORT 8080
```

Comment:
I suggest we demonstrate this using a `docker run` command like above:

```console
$ docker run --name my-running-varnish -e "VARNISH_PORT=80" -d -p 80:80 my-varnish
```

(Also, a side note for you: we don't have `VARNISH_DAEMON_OPTS` but `VARNISHD_PARAMS`. I agree the name is wrong...)